.github/workflows/main.yml

```diff
@@ -5,12 +5,12 @@ name: Build
 on:
   push:
     branches:
-      - main
+      - survey-respondent-help-journey
```

Comment:
I'd leave `main` is place as well. It can be merged into main with the `survey-respondent-help-journey` added as a second branch (so we don't need to keep changing it in upcoming PRs). Also any changes that we _do_ pull in from main won't require conflict merging.